### PR TITLE
Tc04_Redirected to Book Ticket page after logging in

### PR DESCRIPTION
**Pull request by taile47711**

### CODE REVIEW

File: src/test/java/tests/LoginTest.java [hunk 1]
```diff
@@ -38,6 +39,42 @@ public void tc02_UserCantLoginWithBlankUsernameTextbox() {
         String actual = loginPage.getErrorMessage();
 
         LogHelper.info("Verify that user can't login with blank Username textbox");
+
+        Assert.assertEquals(actual, errorMsg, "An error message is not displayed as expected result");
+    }
+
+    @Test(description = "User cannot log into Railway with invalid password")
+    public void tc03_UserCannotLogIntoRailwayWithInvalidPassword() {
+        LogHelper.info("Click tab login");
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTest.java]: Change to `click login tab` instead

Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: fixed

File: src/test/java/tests/LoginTest.java [hunk 2]
```diff
@@ -4,15 +4,16 @@
...
 import page_object.LoginPage;
 
 public class LoginTest extends BaseTest {
     LoginPage loginPage = new LoginPage();
     String errorMsg = "There was a problem with your login and/or errors exist in your form.";
 
-    @Test(description = "User can log into Railway with valid username and password")
-    public void tc001_UserCanLogIntoRailwayWithValidUsernameAndPassword() {
-        LogHelper.info("Click tab login");
+    @Test(description = "User can logging into Railway with valid username and password")
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTest.java]: It's `login` not `logging`

**Review by MinhQuan119 — APPROVED** [review]

File: src/test/java/tests/LoginTest.java (hunk 2 above)
Comment (taile47711) [reply src/test/java/tests/LoginTest.java]: fixed

**Review by tanmle — APPROVED** [review]

**Pull request merged**